guide/private-key-management/cloud-backup.md

```diff
@@ -67,25 +71,31 @@ To see what a user experience with this scheme could look like, see the [daily s
 
 ### Best practice
 
-**When to use**
+#### When to use
 - When risk of self-inflicted loss is higher than loss from theft
 - For beginners, or products with progressive security
 - For wallets with frequent transactions of smaller amounts
 
-**When not to use**
+#### When not to use
 - When storing high values
 - When risk of loss from theft is higher than self-inflicted loss
 - When complete self-sovereignty is important to the target audience
 
-**Variations**
+#### Variations
 - Storage location (keychain, iCloud, Google Drive, One Password etc.)
 - Additional user authentication (biometrics, pin)
 
-**Products that use this scheme**
+#### Products that use this scheme
 - [Photon SDK](https://photonsdk.org) - for use by other wallets, in development
 - [Casa](https://keys.casa)
 - [Pine](https://pine.pm ) - Lightning, beta
 
+{% include tip/open.html label="Encrypted Cloud Backups" %}
+
+You can provide an extra layer of security for the user's private key by encrypting it before sending it to the cloud. One option would be for the user to choose an encryption passphrase. Another would be to have a third-party hold the encryption key so that neither the cloud provider nor the encryption key holder would have access to all the information to use the private key. See [this article](https://blog.keys.casa/casa-keymaster-security-mobile-key-overview/) for an example.
```

Comment:
Encryption is already mentioned further up the page under "How it works". Was a little surprised to see it repeated at the bottom as if it hadn't been mentioned before. Encryption should probably always happen, it's just a matter of how the key is generated and managed.

Reply:
@GBKS That's a good point. The reason I included the paragraph is that there is another line on the page under "Cons" that reads "User needs to trust their OS and cloud storage provider".

When I read this, it sounds to me like a plaintext backup where the user is hoping that their cloud provider doesn't swipe their private key. That's why I decided to include a snippet discussing that the cloud backup can be encrypted in a way to where the cloud provider does not need to be trusted.

I just pushed a new commit where I tweaked the way this is worded and tried to group the tip box closer to the first mention of "encrypted".